backend/src/main/java/com/ody/route/dto/DistanceMatrixResponse.java

```diff
@@ -0,0 +1,39 @@
+package com.ody.route.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+/**
+ * DistanceMatrixResponse 가이드 문서
+ * @link <a href="https://developers.google.com/maps/documentation/distance-matrix/distance-matrix?hl=ko#DistanceMatrixResponse"></a>
+ *
+ * DistanceMatrixStatus 가이드 문서
+ * @link <a href="https://developers.google.com/maps/documentation/distance-matrix/distance-matrix?hl=ko#DistanceMatrixStatus"></a>
+ *
+ * DistanceMatrixElementStatus 가이드 문서
+ * @link <a href="https://developers.google.com/maps/documentation/distance-matrix/distance-matrix?hl=ko#DistanceMatrixElementStatus"></a>
```

Comment:
[제안]
인텔리제이 상에서 아래처럼 보이던데, 개행 수정하면 어떤가요?
![image](https://github.com/user-attachments/assets/c5bb6597-8437-4cad-ade2-35c10a68bc00)

```suggestion
 * DistanceMatrixResponse 가이드 문서 @link <a href="https://developers.google.com/maps/documentation/distance-matrix/distance-matrix?hl=ko#DistanceMatrixResponse"></a><br>
 *
 * DistanceMatrixStatus 가이드 문서 @link <a href="https://developers.google.com/maps/documentation/distance-matrix/distance-matrix?hl=ko#DistanceMatrixStatus"></a><br>
 *
 * DistanceMatrixElementStatus 가이드 문서 @link <a href="https://developers.google.com/maps/documentation/distance-matrix/distance-matrix?hl=ko#DistanceMatrixElementStatus"></a>
```

Reply:
반영 완!